test/acceptance/test/utils.go

```diff
@@ -7,13 +7,14 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
-	"github.com/weaveworks/weave-gitops/pkg/utils"
 	"math/rand"
 	"os"
 	"os/exec"
 	"strings"
 	"time"
 
+	"github.com/weaveworks/weave-gitops/pkg/utils"
+
```

Comment:
Given these are black-box tests, should avoid importing internal packages. Also, I think it may not be needed 🤔

Reply:
I did not add it, it was just moved from the top to that place